### ISSUE TITLE
question about data process

### ISSUE THREAD

**Issue by hustcxx** [ChenxinAn-fdu/CoLo#3]
The paper describes using the [doc] token to generate a representation of the document, but I don't see where to add the [doc] token in the data processing file ext_lable_and_tokenize.py ?

**Comment by ChenxinAn-fdu** [comment]
We used the \<s\> token (for bart) to init the [doc] token in our code.

**Comment by hustcxx** [comment]
Thanks for your response.  Another question. I want to know how did you process the PudMed dataset, like Match???

**Comment by ChenxinAn-fdu** [comment]
yes!  we tokenize the dataset and keep the `max_input_length` to 512.
